packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.scss

```diff
@@ -98,14 +98,20 @@ button.va-crisis-line {
   padding: 8px;
 }
 
-.va-crisis-line-arrow {
-  height: 13px;
-  width: 16px;
-  padding-left: 8px;
-  padding-right: 8px;
+// Right-arrow on modal 'trigger'
+.va-icon__right-arrow {
+  position: relative;
+  top: 2px;
+}
 
+// Modal icons
+.va-clm__icon {
```

Comment:
Just a very minor nit but since the component styles are encapsulated, we probably don't need to namespace the classes like this.

Reply:
I personally believe namespacing the classes is still a best practice, even with encapsulation, as it helps to make it easier to find those classes' definitions in the css files. We namespace classes in lots of other places too, FWIW.

Reply:
>  We namespace classes in lots of other places too

This is probably a symptom of not ever having team standards around naming conventions and folks doing whatever they are used to doing over the years but it's not a big deal. 🤷🏻‍♀️